test/instrumentation/testdata/documentation.md

```diff
@@ -0,0 +1,325 @@
+---
+title: Kubernetes Metrics
+content_type: instrumentation
+---
+
+
+## Metrics
+
+These are the metrics which are exported in Kubernetes components (i.e. kube-apiserver, scheduler, kube-controller-manager, kube-proxy, cloud-controller-manager). 
+
+(auto-generated 2022 Oct 25)
```

Comment:
If this file isn't automatically kept current / tested for staleness, should we commit it?

Reply:
My plan was to run this each code freeze.

Reply:
I'm wary of this.

>  If a piece of information doesn't have to be correct for the system to work, sooner or later it won't be.

- https://utcc.utoronto.ca/~cks/space/blog/sysadmin/SysadminAphorismII

Reply:
I fail to see how omitting this is better than what I am proposing.

Reply:
What I'd do: signpost readers to the document for the latest release (which will form part of the docs on the website). If anyone wants to know how that's different from what's on the _master_ branch, instead signpost them to the script that lets them find out the very latest page.

I really have seen a lot of stale docs in my career, and they nearly all started out with great intentions to keep them current.

Reply:
Website docs PR: https://github.com/kubernetes/website/pull/37522

Reply:
I am happy to abandon my pursuit of documenting metrics if the process to do so exceeds my patience to pursue this.

Reply:
(it's OK to resolve this feedback)